ajax/libs/jclic/package.json

```diff
@@ -0,0 +1,42 @@
+{
+  "name": "jclic",
+  "title": "JClic.js",
+  "description": "HTML5 player for JClic activities",
+  "version": "0.1.43b",
+  "homepage": "http://projectestac.github.io/jclic.js",
+  "readmeFilename": "README.md",
+  "author": "Francesc Busquets <francesc@gmail.com>",
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/projectestac/jclic.js.git"
+  },
+  "keywords": [
+    "jclic",
+    "JClic",
+    "java",
+    "html5",
+    "education",
+    "game",
+    "games",
+    "puzzle",
+    "puzzles",
+    "activities"
+  ],
+  "licenses": [
+    {
+      "type": "EUPL-1.1",
+      "url": "https://spdx.org/licenses/EUPL-1.1.html"
+    }
+  ],
+  "license": "EUPL-1.1",
+  "npmName": "jclic",
+  "npmFileMap": [
+    {
+      "basePath": "src",
+      "files": [
+        "**/*"
+      ]
+    }
+  ],
+  "filename": "activities"
```

Comment:
`"filename"` should point to the minified main file. Let's modify it to `"jclic.min.js"`